src/main/netplay.c

```diff
@@ -382,7 +384,7 @@ static void netplay_send_input(uint8_t control_id, uint32_t keys)
     UDPpacket *packet = SDLNet_AllocPacket(11);
     packet->data[0] = UDP_SEND_KEY_INFO;
     packet->data[1] = control_id; //player number
-    SDLNet_Write32(l_cin_compats[control_id].netplay_count, &packet->data[2]); // current event count
+    SDLNet_Write32(l_cin_compats[control_id].netplay_count + input_delay, &packet->data[2]); // current event count
```

Comment:
When input_delay is a positive int, this change means that the netplay client will not send inputs for the first few frames. In my testing with with m64p's netplay server, this has seemed fine, but I'm not sure if it's something that should be accounted for here.